src/metabase/driver/util.clj

```diff
@@ -31,6 +31,7 @@
       ;; actually if we are going to `throw-exceptions` we'll rethrow the original but attempt to humanize the message
       ;; first
       (catch Throwable e
+        (log/info (trs "Database connection error: {0}" (.getMessage e)))
```

Comment:
This should be `log/error` so if the person sets the default log level to error it still shows up

Reply:
You might as well just log the whole thing so the stacktrace and the class of the Exception is logged too, e.g.

```clj
(log/error e (trs "Database connection error"))
```